### PR TITLE
style(data-picker): add time-picker scrollbar styles

### PR DESCRIPTION
**Pull request by GeorgeHcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

Describe the problem and the scenario：修改时间选择器滚动条样式

修改之前：
image
修改之后
in chrome：
image
in firefox：
image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Modify the time picker scroll bar style       |
| 🇨🇳 Chinese |     修改时间选择器滚动条样式      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at fa54a0c</samp>

Add custom webkit scrollbar styles for date picker panel in dark mode. This enhances the visual consistency and user experience of the `components/date-picker` component.

### <samp>🤖 Generated by Copilot at fa54a0c</samp>

* Add custom styles for the webkit scrollbar of the date picker panel ([link](https://github.com/ant-design/ant-design/pull/45478/files?diff=unified&w=0#diff-a4f33cb3f0dfef43aa860e509b2c43fc62168efd797b258ed2cf47dfc4b1eaa4R923-R936))

### CODE REVIEW

File: components/date-picker/style/index.ts [hunk 1]
```diff
@@ -920,6 +920,20 @@ export const genPanelStyle = (token: SharedPickerToken): CSSObject => {
           transition: `background ${motionDurationMid}`,
           overflowX: 'hidden',
 
+           '&::-webkit-scrollbar': {
```
Comment (kiner-tang) [inline components/date-picker/style/index.ts:923]: 这个应该会有兼容性问题，在火狐上就不行了。要提供一下兼容方案。
image

**Review by kiner-tang — COMMENTED** [review]
可以把优化前后的样式对比图贴到 pr 的内容中

File: components/date-picker/style/index.ts (hunk 1 above)
Comment (GeorgeHcc) [reply components/date-picker/style/index.ts]: firefox里面好像设置不了圆角的样式

**Comment by li-jia-nan** [comment]
感觉 antd 里面很多组件都需要适配一下 win 系统，原生的滚动条在 win 上面太丑了

**Review by kiner-tang — APPROVED** [review]

**Comment by li-jia-nan** [comment]
这些值不需要换成 token 吗？是不是需要考虑一下暗黑模式？

**Comment by MadCcc** [comment]
要用 token。另外滚动条样式应当统一，和虚拟列表的滚动条一致

**Pull request merged**